### PR TITLE
Phase I: Preperation for JNK1, PTP1b, EGFR and Factor Xa

### PR DESCRIPTION
**Pull request by RiesBen**
## Input submission checklist

* [x] created a directory for each system based on the [template directory][templates]
* [x] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [x] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [x] added a PDB file with the protein named `protein.pdb`
* [x] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [x] kept cystallographic waters and metals in the PDB file
* [x] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [x] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

**Comment by RiesBen** [comment]
@hannahbaumann : oh will look into the protonations, thanks for pointing it out! :)

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thank you for making the changes, the ligand files look great and the protonation states of the protein as well! I noticed that for JNK1 (the dataset in the charge annihilation set) you may have copied over the wrong protein (it says EGFR). Could you update that? Also, for Factor Xa, I noticed that your structures has fewer Xray waters than the original input structure. Would it be possible to retain all waters?

Just as a note for later: In EGFR, ligand 27 is present with two "pseudo"(MD) stereoisomers (due to the charge). If we keep both versions, we may have to modify the analysis.

**Comment by RiesBen** [comment]
@hannahbaumann
ups! yeah, you are right, I will fix the jnk1 accident and try to keep all the waters (somehow the prep workflow did remove them after a certain threshold)

Todos:
- [ ] keeping waters
- [ ] fixing JNK1

**Comment by hannahbaumann** [comment]
Thank you @RiesBen ! I looked through it again and I noticed two things that I didn't see before, so sorry about that!
- JNK1 (charge annihilation set) residue 325 (TYR): The carbonyl O for some reason doesn't belong to this residue, but is assigned to residue 553. I have no idea if this would cause any issues, but could you possibly change that?
- JNK1 (JACS set): There are some duplicate ligands in the file with flipped binding modes. Could you remove the ones that are less favorable and document the steps in the preparation details file?

**Comment by RiesBen** [comment]
@hannahbaumann 
hui, updated the files :)

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/PREPARATION_DETAILS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+1. Protein structure was prepared following the maestro instructions: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/pull/89. Afterward NMA cappings were
+converted to NME. I had to add the capping manually for PRO 554, and every chain terminus after.
+2.The structure did not contain any cofactors and no non natural amino acids were found,
+therefore no modifications.
+2. Ligands were left unchanged, no protonation or conformational states were detected.
```
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/PREPARATION_DETAILS.md:5]: Could you update the ligand prep instructions to reflect that you only kept a single binding mode?

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thank you @RiesBen , just the tiny comment to update the ligand prep instructions.

**Review by hannahbaumann — APPROVED** [review]

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+1.  Protein structure was prepared following the maestro instructions: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/pull/89. Afterward NMA cappings were converted to NME.
+2.The structure did not contain any cofactors and no non-natural amino acids were found,
+therefore no modifications. 
+3. Ligands were reduced to the used ligands in: 10.1021/ja512751q;structures were left unchanged left unchanged, no conformational states were detected, but protonations changes
```
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md:4]: There's two instance of ligand 27 in the SDF file, and R and S enantiomer - is this expected?

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+1.  Protein structure was prepared following the maestro instructions: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/pull/89. Afterward NMA cappings were converted to NME.
```
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md:1]: Protonation states haven't changed, but all the protons were replaced it seems - could you confirm that this is the default behaviour with the instructions' prepwizard pre-process step?

Comment (IAlibay) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md:1]: By comparison the protons weren't re-assigned in jnk1.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb:1]: The termini at ASN 8 isn't capped in the schrodinger structure, i.e. it has an NH3, should it be capped?

Same thing for HIS 365 and PHE 563.

I _think_ the same for PRO 554.

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/protein.pdb:1]: Same questions as the charge annihilation set re: capping.

File: industry_benchmarks/input_structures/prepared_structures/scaffold_hopping_set/factor_xa/protein.pdb
Comment (IAlibay) [inline industry_benchmarks/input_structures/prepared_structures/scaffold_hopping_set/factor_xa/protein.pdb:1]: ILE 16 has an NH3 (uncapped) end, in the prepped PDB you add an ACE cap, should it be there?
Same question with LYS243, THR85, and GLU 138.

**Review by IAlibay — CHANGES_REQUESTED** [review]
Thanks @RiesBen - couple of questions to clarify before merge.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb:1]: True, however the maestro approach autmagically caps all terminus structures. In our case here it has a hickup with the proline, therefore I did it partially manually. All our structures were capped by this approach.

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/protein.pdb
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/protein.pdb:1]: see above

File: industry_benchmarks/input_structures/prepared_structures/scaffold_hopping_set/factor_xa/protein.pdb
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/scaffold_hopping_set/factor_xa/protein.pdb:1]: This is the same as above, right?

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb:1]: this is a similar outcome, as in the literature https://pubs.acs.org/doi/suppl/10.1021/ja512751q/suppl_file/ja512751q_si_001.pdf (they use the same workflow)

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md (hunk 3 above)
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md:1]: here you mean the Atom type names, right? :)

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md (hunk 2 above)
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md:4]: they were present here too, so I hope this is korrekt, right? (apparently makes no difference ) https://github.com/schrodinger/public_binding_free_energy_benchmark/blob/main/21_4_results/ligand_predictions/charge_annhil/egfr_pkacorr_out.csv

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md (hunk 3 above)
Comment (IAlibay) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md:1]: No the protons were fully reassigned - i.e. all their coordinates subtly changed.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md (hunk 2 above)
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md:4]: kept only the R  variant -> should resolve this.

File: industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/jnk1/protein.pdb:1]: removed caps  -> should resolve this.

File: industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/protein.pdb
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/jacs_set/jnk1/protein.pdb:1]: removed caps  -> should resolve this.

File: industry_benchmarks/input_structures/prepared_structures/scaffold_hopping_set/factor_xa/protein.pdb
Comment (RiesBen) [reply industry_benchmarks/input_structures/prepared_structures/scaffold_hopping_set/factor_xa/protein.pdb:1]: removed caps  -> should resolve this.

**Review by IAlibay — APPROVED** [review]
This looks good to me, but can you update the preparation details for EGFR @RiesBen ?

**Pull request merged**